src/components/ContactContent/index.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
 
-import { withStyles, Typography, Grid, Link, Icon } from '@material-ui/core';
+import { withStyles, Typography, Grid, Icon } from '@material-ui/core';
 
+import A from '../A';
```

Comment:
`hurumap-ui` has this component `A`

Reply:
Let me see if I can reuse it.

Reply:
Oh.. I remembered why I didn't use it... We haven't integrated `hurumap-ui` just yet... I think that's what @KhadijaMahanga is going to work on next.